README.rst

```diff
@@ -17,11 +17,11 @@ This repository contains a course and some libraries that you can import into yo
      - Download
      - Source OLX
    * - *Open edX demo Course*
-     - ``course-v1:demo+Course+1``
+     - ``course-v1:Axim+DemoX+demo_course``
```

Comment:
```suggestion
     - ``course-v1:Open_edX+DemoX+demo_course``
```

Maybe? I think I want to position this as the community's demo course, not Axim's